smtm/controller/telegram/telegram_controller.py

```
@@ -45,14 +46,15 @@ def main(self, exchange="UPB", currency="BTC", budget=500000) -> None:
             "strategy": "BNH",
             "strategy_files": ["sma_crossover.md", "rsi_strategy.md", "buy_and_hold.md"],
         }
-        self.operator = SystemOperator(llm_client, config)
+        self.operator = SystemOperator(llm_client, config,
+                                       account_store=AccountStore())
         try:
             self.operator.setup()
-        except ValueError as err:
+        except Exception as err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)